host/sampi/http.go

```diff
@@ -53,15 +58,19 @@ func (s *Cluster) AddJobs(req map[string][]*host.Job) (map[string]host.Host, err
 }
 
 // Host Service methods
-
 func (s *Cluster) RemoveJobs(hostID string, jobIDs ...string) error {
+	l := s.logger.New("fn", "RemoveJobs", "host.id", hostID)
+	l.Debug("beginning to remove jobs")
 	s.state.Begin()
+	s.logger.Debug("now removing jobs from host")
 	s.state.RemoveJobs(hostID, jobIDs...)
 	s.state.Commit()
 	return nil
 }
 
 func (s *Cluster) StreamHostEvents(ch chan host.HostEvent, done chan bool) error {
+	l := s.logger.New("fn", "StreamHostEvents")
+	l.Debug("adding host event listener", "at", "add_listener")
```

Comment:
This line isn't useful if we log from the API side, as it has no details about where the listener came from.